templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -72,11 +104,13 @@ const initNavigationSliding = () => {
         element.setAttribute('tabindex', '-1');
       });
     });
-    target.querySelectorAll('li').forEach(function (element) {
-      if (element.parentNode === target) {
-        element.children[0].setAttribute('tabindex', '0');
-      }
-    });
+    if (target) {
+      target.querySelectorAll('li').forEach(function (element) {
+        if (element.parentNode === target) {
+          element.children[0].setAttribute('tabindex', '0');
```

Comment:
Fair question, I haven't really modified it (just wrapped in conditional). And this isn't exactly the scope of this PR (this relates to mobile multi-level "sliding" navigation).

But, answering your question, I believe the intention was to make currently visible level tab-navigatable, not any child levels. So that you don't tab-navigate off screen.
Or at least that's my understanding. I haven't been heavily involved when it was implemented.

But indeed, some parts of this JS would be worth second look and refactoring. But this can be done later (as we have separate task to tackle the sliding nav issues).